.github/workflows/pull_request.yml

```diff
@@ -10,7 +10,7 @@ jobs:
         os: [ubuntu-latest, windows-2016]
         node-version: [10.x, 12.x, 13.x]
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
I’m seeing GitHub Actions starting to recommend `checkout@v2` now (like [setup-node](https://github.com/actions/setup-node)). I don’t think it affects anything, but I changed it here because GitHub Actions kept failing every time on `v1` and this seems to help reliability